discuss/getting_started.rst

```diff
@@ -4,187 +4,13 @@
 Discuss
 =======
 
-Overview
```

Comment:
Moreover, as there is still a reference to getting_started in applications.rst, the doc won't be built I think.